### PR TITLE
Fix dataGenerator, implement Logstash, load to EC2 instance

### PR DESCRIPTION
**Pull request by snackersteph**

### CODE REVIEW

File: database/index.js [hunk 1]
```diff
@@ -24,44 +24,45 @@ sequelize
...
+// profit.sync();
+
 
 // ====================================================== 
 // **************** BULK INSERT METHODS *****************
 // ====================================================== 
 
 const insertUserPackets = (records)=> {
-    MyUsers.bulkCreate(records);
+    user.bulkCreate(records);
```
Comment (kennyxcao) [inline database/index.js]: bulkCreate starts a promise so it should be properly handled using then and catch, so you can check the result of bulkCreate

File: server/dataGenerator.js [hunk 2]
```diff
@@ -38,58 +38,57 @@ myNamespace.round = function(number, precision) {
...
+    let profit_data = [];
+    
+    for (let i = 0; i < rounds; i ++) {
+        let data = makeRandom(array);
+        user_data.push(userPackets(data));
+        indicator_data.push(indicatorPackets(data));
+        profit_data.push(profitPackets(data, array));
+    }
+    
+    database.insertUserPackets(user_data);
```
Comment (kennyxcao) [inline server/dataGenerator.js]: If these bulk inserts have to be in sequential order, try use promises and chain them one after another or use bluebird's Promise.all to run simultaneously then check if they're all fulfilled.
You might get weird results if they are running separately.

File: database/index.js (hunk 1 above)
Comment (snackersteph) [reply database/index.js]: fixed!

File: server/dataGenerator.js (hunk 2 above)
Comment (snackersteph) [reply server/dataGenerator.js]: got it! thanks Kenny!

**Pull request merged**